### PR TITLE
General cleanup and preparation for Travis deployment

### PR DESCRIPTION
**Pull request by kdembler**
Summary of changes:

- Removed files needed for PyPI - plugin is an application, not a library and isn't meant to be distributed by PyPI - instead, releases will be automated with Travis and pushed to GitHub
- General cleanup

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +0,0 @@
-include versioneer.py
```
Comment (mkleina) [inline MANIFEST.in]: This file was autogenerated by `versioneer` command and contains list of files for source distribution:
https://docs.python.org/2/distutils/sourcedist.html

We should discuss whether this file is needed or not.

File: setup.cfg [hunk 2]
```diff
@@ -1,5 +0,0 @@
-[versioneer]
```
Comment (mkleina) [inline setup.cfg]: This file is used by `versioneer` command. It should not be removed if user will want to upgrade versioneer related files.

File: setup.py [hunk 3]
```diff
@@ -1,39 +0,0 @@
-# -*- coding: utf-8 -*-
```
Comment (mkleina) [inline setup.py]: This file is needed by `make pkg` command:
```bash
_info "virtual environment: installing plugin package"
$PYTHON_ENV/bin/pip install -I ${__proj_dir}
```

without this file, ACI package generation is impossible.

**Review by mkleina — CHANGES_REQUESTED** [review]

File: setup.py (hunk 3 above)
Comment (kdembler) [reply setup.py]: You're right, I missed that. Changed PR to WIP, will fix this.

**Comment by kdembler** [comment]
Following some discussion, approach from PR description is no longer accurate. We decided to  deploy Python plugins to PyPI to give user additional installation option.

**Review by mkleina — APPROVED** [review]

**Pull request merged**